micronaut-camunda-bpm-feature/transactional-test-data-jdbc/src/test/java/info/novatec/micronaut/camunda/bpm/feature/test/BookRepository.kt

```diff
@@ -0,0 +1,14 @@
+package info.novatec.micronaut.camunda.bpm.feature.test
+
+import io.micronaut.data.jdbc.annotation.JdbcRepository
+import io.micronaut.data.model.query.builder.sql.Dialect
+import io.micronaut.data.repository.CrudRepository
+
+/**
+ * Sample repository for micronaut-data.
+ *
+ * @author Tobias Schäfer
+ */
+@JdbcRepository(dialect = Dialect.H2) interface BookRepository : CrudRepository<Book?, Long?> {
+    fun countByTitle(title: String?): Long
```

Comment:
That would make the nullable type here probably also obsolete.

Reply:
thanks - I removed the "?"